5.3/5.3.4/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 5.3.4
+Напишите функцию checkSpam(str), возвращающую true, если str содержит 'viagra' или 'XXX', а иначе false.
+Функция должна быть нечувствительна к регистру:
+checkSpam('buy ViAgRA now') == true
+checkSpam('free xxxxx') == true
+checkSpam("innocent rabbit") == false
+<script>
+	function checkSpam(str) {
```

Comment:
Let's think will the condition `Boolean(str.toUpperCase().includes('hello'))` ever return `true`?
or for example, will the following condition ever return `true`:
`Boolean(str.toLowerCase().includes('HELLO'))`

the `str` variable can content any value